app/src/main/java/com/spielpark/steve/bernieapp/fragments/ConnectFragment.java

```diff
@@ -49,12 +56,30 @@
  */
 public class ConnectFragment extends Fragment {
     private static ConnectTask mTask;
-    private static GoogleMap map;
+    private GoogleMap map;
     private HashMap<Marker, Integer> mHashMap;
     public String mZip = "";
     public int mRadius = 50;
     public boolean fetchCountry = true;
     private static ConnectFragment mInstance;
+
+    @Bind(R.id.c_btnGo) Button goButton;
```

Comment:
Not a big deal but annotations before code doesnt match the code style of existing code that uses them above members